app/views/provider_interface/application_choices/show.html.erb

```diff
@@ -25,6 +25,16 @@
   ) %>
 </p>
 
+<% if FeatureFlag.active?(:new_references_flow_providers) %>
+  <p class="govuk-body govuk-!-display-none-print">
+    <%= govuk_link_to(
+      'Download references (PDF)',
+      provider_interface_application_choice_references_path(@application_choice.id, format: :pdf),
+      download: "#{@application_choice.application_form.support_reference}-references",
+    ) %>
+  </p>
+<% end %>
```

Comment:
I wonder if it might be better to have this link at the top of the references tab instead? (That way it’d be more obvious what content will be included as you’d already be looking at the page)

Reply:
makes sense!